backend/app/models/article.rb

```diff
@@ -19,6 +19,7 @@ class Article < ActiveRecord::Base
   after_save :update_published_articles_count
 
   dragonfly_accessor :cover do
+    after_assign {|img| self.cover = img.image_optim }
```

Comment:
So the problem with doing it this way I believe is that it would run on the main thread blocking the thread from serving any users - slowing the site for everybody. 

I could be wrong though so maybe it's worth testing this. Wanna work on the staging instance of Manshar first and try to deploy this to it?